examples/working-directly-with-json/working_directly_with_json.bal

```diff
@@ -10,12 +10,12 @@ json j = {
     }
 };
 
-// Field access is allowed on the `json`-typed variable. However, the return
-// type would be a union of `json` and `error`.
+// Field access is allowed on the `json` typed variable. However, the return type would be a union
+// of `json` and `error`.
 json v = check j.x.y;
 string s1 = check v.z;
 
-// `ensureType` method can also be used to perform conversions.
+// `ensureType()` method can also be used to perform conversions.
```

Comment:
```suggestion
// The `ensureType()` method can also be used to perform conversions.
```

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-distribution/pull/2930/commits/850aaadbbb3db21491d1ca85c58cf87a6dc589e8.